draft-ietf-quic-multipath.md

```diff
@@ -902,68 +879,23 @@ Path ID (e.g. more than half of the available Path ID are used).
+As stated in {{introduction}}, when the multipath extension is negotiated, each
 path uses a separate packet number space.
 This is a major difference from
 {{QUIC-TRANSPORT}}, which only defines three number spaces (Initial,
 Handshake and Application packets).
 
-Implementation of multipath capable QUIC will need to carefully
-model the relations between paths and number spaces, as shown
-in {{fig-number-spaces}}.
-
-~~~
-   +-------------------------+
-   | CID received from peer: |-- - - - - - +
-   | Sender Number Space     |             |
-   +-------------------------+             v
-                      ^             +----------------+
-                      |             | Path (4 tuple) |
-                      +-------------| - RTT          |
-                +------------------>| - Congestion   |
-                |                   |   state        |
-                v                   +----------------+
-   +-------------------------+             ^
-   | CID provided to peer:   |             |
-   | Receiver Number Space   |-- - - - - - +
-   +-------------------------+
-~~~
-{: #fig-number-spaces title="Send and Receive number spaces"}
-
-The path is defined by the 4-tuple through which packets are
-received and sent. Packets sent on the path will include the
-Destination Connection ID currently used for that path, selected
-from the list of connection IDs provided by the peer. Packets received
-on the path carry a Destination CID selected by the peer from
-the list provided to that peer.
-
 The relation between packet number spaces and paths is fixed.
-CIDs are separately allocated for each Path ID.
+Connection IDs are separately allocated for each Path ID.
 Rotating the connection ID on a path does not change the Path ID.
 NAT rebinding, though it changes the 4-tuple of the path,
 also does not change the path identifier.
 The packet number space does not change when connection ID
 rotation happens within a given Path ID.
 
-Data associated with the transmission and reception on a given
-path can be associated to either the "path state", or to the
-state of either the sender or receiver number spaces. For example:
-
-* RTT measurements and congestion state are logically associated
-  with the 4-tuple. They will remain unchanged if data starts
-  being received or sent through the same 4-tuple using new
-  connection IDs.
-
-* Implementations of loss recovery typically maintain lists of
-  packets sent and not yet acknowledged. Such information, along
-  with the value of the next PN to use for sending, is
-  logically associated with the "Sender Number Space", which remain
-  unchanged when connection ID rotation happens.
-
-* Sending of acknowledgement requires keeping track of the PN of
-  received packets and of acknowledgements previously sent. Such
-  information is logically associated with the "Receiver Number Space",
-  which remain unchanged when connection ID rotation happens.
```

Comment:
This was removed as per-packet number space and per path space is now the same. We might want to say something else instead. See issue #360

Maybe we can even remove the whole section?